conf/asapo_transfer_config.yaml

```diff
@@ -0,0 +1,38 @@
+# ASAPO produces endpoint
+endpoint: localhost:8000
+
+# ASAPO produces beamtime
+beamtime: auto
+
+# ASAPO produces token
+token: bla
+
+# ASAPO data_source
+default_data_source: hidra_test
+
+# Number of threds for ASAPO producer
+n_threads: 1
+
+# Starting file index
+start_file_idx: 1
+
+# Template to file path, which includes `stream` and `file_idx`
+file_regex: "current/raw/(?P<scan_id>.*)_(?P<file_idx_in_scan>.*).h5"
+
+# ASAPO send timeout in [s]
+timeout: 30
+
+#Signal host
+signal_host: 'localhost'
+
+#Target port
+target_port: '50101'
+
+#Target directory
+target_dir: ''
+
+#Timeout to reconnect to sender
+reconnect_timeout: 3
+
+#Set log level for the application
+log_level: "DEBUG"
```

Comment:
`beamtime`, `n_threads`, `start_file_idx`, , `file_regex`, `timeout`, `target_dir`, `reconnect_dir` and `log_level` should have defaults so that they don't need to be mentioned in the config file. 

The following values should be determined automatically if not given in config file
- `default_data_source` can be constructed from the `detector_id`
- `signal_host` can be looked up via the `beamline` from the `hidra._constants` module
- `target_port` should stay for this PR but will be replaced with a following PR by me

It is common to leave the default values in the example config file but comment them out for documentation.

Reply:
Default values in the config file is commented out. Values are defined in the Config class inside asapo_transfer.py. I don't understand, how `hidra._constants` is populated. Currently, there is not much information inside.